### PR TITLE
Prepare for 12.4.0~pre1 prerelease

### PR DESCRIPTION
**Pull request by WilliamLewww**
# 🎈 Prerelease

Preparation for 12.4.0~pre1 prerelease.

Comparison to 12.3.0: https://github.com/ignitionrobotics/sdformat/compare/sdformat12_12.3.0...sdformat12

Needed by:
* https://github.com/ignitionrobotics/ign-sensors/pull/194

## Checklist
- [ ] Asked team if this is a good time for a release
- [ ] There are no changes to be ported from the previous major version
- [ ] No PRs targeted at this major version are close to getting in
- [ ] Bumped minor for new features, patch for bug fixes
- [ ] Updated changelog
- [ ] Updated migration guide (as needed)
- [ ] Link to PR updating dependency versions in appropriate repository in [ignition-release](https://github.com/ignition-release) (as needed): <LINK>

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,207 @@
...
+    * [Pull request #Merge branch 'sdf10' into merge_10_11](https://github.com/ignitionrobotics/sdformat/pull/Merge branch 'sdf10' into merge_10_11)
+
+1. Merge pull request #807 from scpeters/merge_9_to_11
+    * [Pull request #807 from scpeters/merge_9_to_11](https://github.com/ignitionrobotics/sdformat/pull/807 from scpeters/merge_9_to_11)
+
+1. sdf_custom: fix nested model expectations
+    * [Pull request #807](https://github.com/ignitionrobotics/sdformat/pull/807)
+
+1. Merge commit '6b53f954b8b74d0bdba33982d59c25da8b280ee4' into merge_9_to_11
+    * [Pull request #Merge commit '6b53f954b8b74d0bdba33982d59c25da8b280ee4' into merge_9_to_11](https://github.com/ignitionrobotics/sdformat/pull/Merge commit '6b53f954b8b74d0bdba33982d59c25da8b280ee4' into merge_9_to_11)
```
Comment (chapulina) [inline Changelog.md:122]: We should remove these merge commits from the changelog

File: Changelog.md [hunk 2]
```diff
@@ -1,5 +1,207 @@
...
+    * [Pull request #780](https://github.com/ignitionrobotics/sdformat/pull/780)
+
+1. 9 ➡️ 10
+    * [Pull request #795](https://github.com/ignitionrobotics/sdformat/pull/795)
+
+1. Support printing sdf poses in degrees and allow snapping to commonly used angles
+    * [Pull request #689](https://github.com/ignitionrobotics/sdformat/pull/689)
+
+1. 9 ➡️  10
+    * [Pull request #9 ➡️  10](https://github.com/ignitionrobotics/sdformat/pull/9 ➡️  10)
```
Comment (chapulina) [inline Changelog.md:146]: The forward-ports can also be ommitted, because the PRs that were actually ported should already show up as separate items

**Review by chapulina — COMMENTED** [review]
Thanks! I just have some comments about cleaning up the changelog.

This release should also help with https://github.com/ignitionrobotics/ign-gazebo/pull/1333

File: Changelog.md [hunk 3]
```diff
@@ -1,5 +1,207 @@
 ## libsdformat 12.X
 
+### libsdformat 12.4.0 (2022-01-xx)
+1. Add SDFormat tags for Triggered Cameras
+    * [Pull request #846](https://github.com/ignitionrobotics/sdformat/pull/846)
+
+1. sdf -> usd: Added sensor
```
Comment (scpeters) [inline Changelog.md:7]: I would cluster the USD pull requests together as well

**Comment by WilliamLewww** [comment]
Converted to draft to wait for https://github.com/ignitionrobotics/sdformat/pull/831 functionality on pre-release.

**Comment by nkoenig** [comment]
Just a reminder that the next sdf12 full release should have this PR in it: https://github.com/ignitionrobotics/sdformat/pull/842.

**Comment by scpeters** [comment]
> Just a reminder that the next sdf12 full release should have this PR in it: #842.

this is just a prerelease; we'll check again before making a stable release

**Comment by adlarkin** [comment]
> Converted to draft to wait for #831 functionality on pre-release.

#831 has been merged, and I also just realized that we aren't releasing binaries for the USD component yet, so feel free to make the pre-release whenever it's needed :+1:

**Review by nkoenig — APPROVED** [review]

**Pull request merged**